trunk/includes/pardot-api-class.php

```diff
@@ -16,19 +17,20 @@
  * @version 1.0.0
  *
  */
-class Pardot_API {
+class Pardot_API
+{
 	/**
 	 * @const string The root URL for the Pardot API.
 	 *
 	 * @since 1.0.0
 	 */
-	const ROOT_URL = 'https://pi.pardot.com/api';
+	const API_ROOT_URL = Pardot_Settings::PI_PARDOT_URL . '/api';
```

Comment:
I also updated the name of this constant to be a bit more descriptive since it isn't really the "root" URL for all of Pardot, but instead just for the API calls